Test failed on apple silicon

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
-->


**What happened**:
I'm doing it while looking at https://github.com/yorkie-team/yorkie#developing-yorkie.
I tested it on apple silicon, but it failed.
```
go clean -testcache
go test -tags integration -race ./...
?       github.com/yorkie-team/yorkie/api       [no test files]
ok      github.com/yorkie-team/yorkie/api/converter     0.293s
ok      github.com/yorkie-team/yorkie/client    0.305s
?       github.com/yorkie-team/yorkie/cmd/yorkie        [no test files]
?       github.com/yorkie-team/yorkie/internal/cli      [no test files]
?       github.com/yorkie-team/yorkie/internal/version  [no test files]
ok      github.com/yorkie-team/yorkie/pkg/cache 0.131s
ok      github.com/yorkie-team/yorkie/pkg/document      0.226s
ok      github.com/yorkie-team/yorkie/pkg/document/change       0.221s
# bou.ke/monkey
../../go/pkg/mod/bou.ke/monkey@v1.0.2/replace.go:24:14: undefined: jmpToFunctionValue
ok      github.com/yorkie-team/yorkie/pkg/document/json 0.321s
?       github.com/yorkie-team/yorkie/pkg/document/key  [no test files]
?       github.com/yorkie-team/yorkie/pkg/document/operation    [no test files]
?       github.com/yorkie-team/yorkie/pkg/document/proxy        [no test files]
ok      github.com/yorkie-team/yorkie/pkg/document/time 0.285s
ok      github.com/yorkie-team/yorkie/pkg/llrb  0.311s
ok      github.com/yorkie-team/yorkie/pkg/pq    0.285s
ok      github.com/yorkie-team/yorkie/pkg/splay 0.344s
?       github.com/yorkie-team/yorkie/pkg/types [no test files]
?       github.com/yorkie-team/yorkie/test/helper       [no test files]
ok      github.com/yorkie-team/yorkie/test/integration  12.190s
ok      github.com/yorkie-team/yorkie/yorkie    0.304s
?       github.com/yorkie-team/yorkie/yorkie/auth       [no test files]
ok      github.com/yorkie-team/yorkie/yorkie/backend    0.544s
?       github.com/yorkie-team/yorkie/yorkie/backend/background [no test files]
ok      github.com/yorkie-team/yorkie/yorkie/backend/db 0.199s
FAIL    github.com/yorkie-team/yorkie/yorkie/backend/db/memory [build failed]
ok      github.com/yorkie-team/yorkie/yorkie/backend/db/mongo   0.568s
?       github.com/yorkie-team/yorkie/yorkie/backend/housekeeping       [no test files]
?       github.com/yorkie-team/yorkie/yorkie/backend/sync       [no test files]
ok      github.com/yorkie-team/yorkie/yorkie/backend/sync/etcd  1.623s
ok      github.com/yorkie-team/yorkie/yorkie/backend/sync/memory        0.565s
?       github.com/yorkie-team/yorkie/yorkie/clients    [no test files]
?       github.com/yorkie-team/yorkie/yorkie/log        [no test files]
?       github.com/yorkie-team/yorkie/yorkie/packs      [no test files]
ok      github.com/yorkie-team/yorkie/yorkie/profiling  0.682s
?       github.com/yorkie-team/yorkie/yorkie/profiling/prometheus       [no test files]
ok      github.com/yorkie-team/yorkie/yorkie/rpc        0.927s
?       github.com/yorkie-team/yorkie/yorkie/rpc/interceptors   [no test files]
FAIL
make: *** [test] Error 2

```


**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

1. docker-compose -f docker/docker-compose.yml up --build -d
2. make test 


**Anything else we need to know?**:
go version is go1.17.6 darwin/arm64.

**Environment**:
- Operating system: macOS Big Sur 11.4
- Browser and version: 
- Yorkie version (use `yorkie version`): v0.2.1
- Yorkie JS SDK version:

--- a/yorkie/backend/db/memory/db_test.go
+++ b/yorkie/backend/db/memory/db_test.go
@@ -20,9 +20,7 @@ import (
 	"context"
 	"fmt"
 	"testing"
-	gotime "time"
 
-	"bou.ke/monkey"
 	"github.com/stretchr/testify/assert"
 
 	"github.com/yorkie-team/yorkie/pkg/document"
@@ -180,31 +178,5 @@ func TestDB(t *testing.T) {
 		snapshot, err = memdb.FindLastSnapshotInfo(ctx, docInfo.ID)
 		assert.NoError(t, err)
 		assert.Equal(t, uint64(1), snapshot.ServerSeq)
-	})
-
-	t.Run("housekeeping test", func(t *testing.T) {
-		ctx := context.Background()
-
-		yesterday := gotime.Now().Add(-24 * gotime.Hour)
-		guard := monkey.Patch(gotime.Now, func() gotime.Time { return yesterday })
-		clientA, err := memdb.ActivateClient(ctx, fmt.Sprintf("%s-A", t.Name()))
-		assert.NoError(t, err)
-		clientB, err := memdb.ActivateClient(ctx, fmt.Sprintf("%s-B", t.Name()))
-		assert.NoError(t, err)
-		guard.Unpatch()
-
-		clientC, err := memdb.ActivateClient(ctx, fmt.Sprintf("%s-C", t.Name()))
-		assert.NoError(t, err)
-
-		candidates, err := memdb.FindDeactivateCandidates(
-			ctx,
-			gotime.Hour,
-			10,
-		)
-		assert.NoError(t, err)
-		assert.Len(t, candidates, 2)
-		assert.Contains(t, candidates, clientA)
-		assert.Contains(t, candidates, clientB)
-		assert.NotContains(t, candidates, clientC)
 	})
 }

--- a/yorkie/backend/db/memory/housekeeping_test.go
+++ b/yorkie/backend/db/memory/housekeeping_test.go
@@ -1,0 +1,62 @@
+//go:build amd64
+
+/*
+ * Copyright 2022 The Yorkie Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package memory_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	gotime "time"
+
+	"bou.ke/monkey"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/yorkie-team/yorkie/yorkie/backend/db/memory"
+)
+
+func TestHousekeeping(t *testing.T) {
+	memdb, err := memory.New()
+	assert.NoError(t, err)
+
+	t.Run("housekeeping test", func(t *testing.T) {
+		ctx := context.Background()
+
+		yesterday := gotime.Now().Add(-24 * gotime.Hour)
+		guard := monkey.Patch(gotime.Now, func() gotime.Time { return yesterday })
+		clientA, err := memdb.ActivateClient(ctx, fmt.Sprintf("%s-A", t.Name()))
+		assert.NoError(t, err)
+		clientB, err := memdb.ActivateClient(ctx, fmt.Sprintf("%s-B", t.Name()))
+		assert.NoError(t, err)
+		guard.Unpatch()
+
+		clientC, err := memdb.ActivateClient(ctx, fmt.Sprintf("%s-C", t.Name()))
+		assert.NoError(t, err)
+
+		candidates, err := memdb.FindDeactivateCandidates(
+			ctx,
+			gotime.Hour,
+			10,
+		)
+		assert.NoError(t, err)
+		assert.Len(t, candidates, 2)
+		assert.Contains(t, candidates, clientA)
+		assert.Contains(t, candidates, clientB)
+		assert.NotContains(t, candidates, clientC)
+	})
+}

